warehouse/ingest-core/src/test/java/datawave/ingest/mapreduce/handler/edge/ProtobufEdgeDeleteModeTest.java

```diff
@@ -243,141 +230,12 @@ public void setup(TaskAttemptContext context) {
         public long process(KEYIN key, RawRecordContainer event, Multimap<String,NormalizedContentInterface> fields,
                         TaskInputOutputContext<KEYIN,? extends RawRecordContainer,KEYOUT,VALUEOUT> context, ContextWriter<KEYOUT,VALUEOUT> contextWriter)
                         throws IOException, InterruptedException {
-            long edgesCreated = 0;
```

Comment:
`this.deleteMode` (above) is never used.  I'm wondering if MyDerivedProtobufEdgeDataTypeHandler could be removed.